From 9c504616e38e6dd37a27aaa50243134cdf1a900c Mon Sep 17 00:00:00 2001
From: Michael Walle <michael.walle@kontron.com>
Date: Fri, 18 Oct 2019 10:17:22 +0200
Subject: [PATCH 03/15] fix parallel build

Signed-off-by: Michael Walle <michael.walle@kontron.com>
---
 Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index fcf6315..3dba069 100644
--- a/Makefile
+++ b/Makefile
@@ -32,13 +32,13 @@ build: $(TSN_LIB) $(TSN_BIN) $(TSN_EVENT) $(TSTAMP_BIN)
 $(TSN_LIB): $(LIB_DEPS)
 	$(CC) -shared $(LIB_OBJ) -o $@ $(LIB_LDFLAGS)
 
-$(TSN_BIN): $(BIN_DEPS)
+$(TSN_BIN): $(BIN_DEPS) $(TSN_LIB)
 	$(CC) $(BIN_OBJ) -o $@ $(BIN_LDFLAGS)
 
-$(TSN_EVENT): tools/$(TSN_EVENT).o
+$(TSN_EVENT): tools/$(TSN_EVENT).o $(TSN_LIB)
 	$(CC) tools/$(TSN_EVENT).o -o tools/$(TSN_EVENT) $(BIN_LDFLAGS)
 
-$(TSTAMP_BIN): tools/$(TSTAMP_BIN).o
+$(TSTAMP_BIN): tools/$(TSTAMP_BIN).o $(TSN_LIB)
 	$(CC) tools/$(TSTAMP_BIN).o -o tools/$(TSTAMP_BIN) $(BIN_LDFLAGS)
 
 lib/%.o: lib/%.c
-- 
2.17.1

